### PR TITLE
doc: add remaining meeting minutes

### PR DESCRIPTION
**Pull request by Starefossen**
Add meeting minutes for the following Build WG Meetings:

* 2015-12-01 (#269)
* 2015-12-22 (#283)
* 2016-01-12 (#300)
* 2016-02-02 (#313)
* 2016-02-23 (#336)
* 2016-03-15 (#351)

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2016-01-12.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Minutes
+
+### Alpine Linux / Docker Build [#75 #79]
+
+Hans: look into getting it into ansible
+
+Hans: get a vm that can run docker and install alpine to avoid vm restrictions
+
+Johnan: start a docker vm at digitalocean to host this.
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-01-12.md]: *Johan

File: doc/meetings/build-wg-meeting-2016-01-12.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Michael: add docker to required packages in one of our ansible hosts
+
+### New readme, new company listing incl logos, new team listing [#294]
+
+find a way to distinct between tiers (possibly number of vm deploys)
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-01-12.md]: Nitpick: Find

File: doc/meetings/build-wg-meeting-2016-03-15.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+Discussion about the scope, longer term mandate for test working group.  Key
+part is that access to jenkins and machines is still through build working group
+and will be worked out between the 2 teams.
+
+No objections. No abstains. All in favor.
+
+### Per-job acl at ci.nodejs.org
+
+We need to define who has access to jenkins and how new jobs/tests can be
+created.  Libuv is an example where we can delegate running/modifying job.
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-03-15.md]: Nitpick: two spaces

**Comment by jbergstroem** [comment]
Generally LGTM; feel free to fix nitpick if you care.

**Comment by mhdawson** [comment]
LGTM, thanks for doing this

**Comment by Starefossen** [comment]
@jbergstroem thanks for going over and finding improvements 👍 I have pushed the updated meeting minutes now.

**Comment by mhdawson** [comment]
@Starefossen you going to land ?

**Comment by Starefossen** [comment]
@mhdawson AFK for a few hours, feel free to land this in the mean time :smile:

**Comment by jbergstroem** [comment]
LGTM. I can land.

**Comment by jbergstroem** [comment]
Merged in c59f87e522401886d817b61ad47fec4ff0370d83!

**Pull request closed without merging**